src/config/chis-tg/config.json

```diff
@@ -0,0 +1,196 @@
+
+{
+  "domains": [
+    {
+      "friendly": "MoH Togo DEV",
+      "domain": "moh-togo.dev.medicmobile.org"
+    },
+    {
+      "friendly": "MoH Togo UAT",
+      "domain": "moh-togo-uat.dev.medicmobile.org"
+    }
+  ],
+  "contact_types": [
+    {
+      "name": "cg70_supervision",
+      "friendly": "Supervision Area",
+      "contact_type": "cg72_supervision",
+      "user_role": ["supervisor"],
+      "username_from_place": true,
```

Comment:
doesn't the username have both the supervisor name and the sanitary training name? do we really want all that in the username? my guess is we want this to be false and just the name of the supervisor will be used